src/InsightContainer/index.js

```diff
@@ -35,8 +36,8 @@ const useStyles = makeStyles(({ breakpoints, variant }) => ({
     marginTop: '1rem'
   },
   highlight: {
-    height: '100%',
     width: '100%',
+    overflow: 'hidden',
```

Comment:
This seems to break source:
![S](https://user-images.githubusercontent.com/1779590/66893194-26980100-eff6-11e9-95d4-2cbeef63eeee.png)